packages/python-awscli/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-awscli
```

Comment:
Remove this, it's already packaged under https://archlinux.org/packages/community/any/aws-cli/